src/components/dashboard/FaceVerification/index.web.js

```diff
@@ -35,6 +35,9 @@ const kindOfCameraIssuesMap = mapValues(
       'VideoCaptureStreamNotActive',
     ],
 
+    // Context
+    ContextError: ['ContextSwitch'],
```

Comment:
can you explain what this is? how does that pass the error/status code to the error screen?

Reply:
Just checked ErrorScreen - it’s unchanged. Probably you haven’t committed something ? @StanislavShevchenko Pls re-check it this morning

Reply:
@sirpy Remember, we had a task to notify user that face verification was failed due to the invalid device orientation. 

As you can see, we’re catching corresponding error codes and passing specific error.name to the ErrorScreen

Could we also do those task (notify user about invalid orientation) In the scope of this task/PR ? I think it’s important for user to understand how he should place device to verify himself

Reply:
> can you explain what this is? how does that pass the error/status code to the error screen?

@sirpy just speaking with @StanislavShevchenko - he told that you asked to add ability to recognize the case when Zoom cancelled session when you switched browser tab. For now - just to add specific flag or something else could be used on ErrorScreen to understand the kind of the issue

As we already has an kingOfIssues map, Stas just have added 'ContextError' kind of the issue with ZoomSessionStatus.ContextSwitch code describing this case. 

This map use used as the following
1. FIrstly, just in declaration, we replacing string keys to theirs values via mapValues/map:
`mapValues(<.. declated map>, statusesKeys => statusesKeys.map(key => ZoomSessionStatus[key]))`
2. Then, on some non-success session status happens, we're matching status code against the map and finding which kindOfTheIsue it corresponds:
`const kindOfTheIssue = findKey(kindOfSessionIssuesMap, statusCodes => statusCodes.includes(status))`
3. Kind of the issue found is added to the error.name property will be then passed to the ErrorScreen
4. on Error screen we could print different messages regarding to the error.name passed

Probably 'ContextError/ContextSwitch' name isn't so clear (it's term from multitasking system programming). I've changed it to ForegoundLoosedError which is more clear from my point of view

Reply:
ok 10x